src/core/ext/transport/inproc/inproc_transport.c

```diff
@@ -1263,8 +1263,8 @@ grpc_channel *grpc_inproc_channel_create(grpc_server *server,
 
   grpc_arg default_authority_arg;
   default_authority_arg.type = GRPC_ARG_STRING;
```

Comment:
While you're at it, this could be changed to use `grpc_channel_arg_string_create()`.